independent-publisher-connectors/AfterShip/apiProperties.json

```diff
@@ -0,0 +1,22 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Key",
+          "description": "The API Key for this api",
+          "tooltip": "Provide your API Key",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#f9a31a",
+    "capabilities": [],
+    "publisher": "Taiki Yoshida"
```

Comment:
@taiki-yoshida The `"stackOwner" property is missing. Please add it.

Docs - [publisher and stackOwner properties](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#step-3a-publisher-and-stackowner-properties)